app/views/qsets/_breadcrumbs.html.erb

```diff
@@ -0,0 +1,10 @@
+<h5 class = "center">
+  <span id="breadcrumbs" class = "center">
+    <% if not @qset.root? %>
+      <% for qset in @qset.ancestors.reverse %>
+        <%= link_to h(qset.name), qset, style: "color:#00b3fe" %> &gt;
```

Comment:
Can we change this so that we add a class that in turn is defined in our CSS styles with a certain style (in this case, a link color)? That way the CSS is the single source of truth for how our app is styled, and it's easier to make changes to styling all in one place.